packages/react-cookie/index.d.ts

```diff
@@ -3,6 +3,13 @@ import * as React from 'react';
 export type Cookie = string;
 
 export class Cookies {
+    constructor(
+        cookieHeader?: string | object,
```

Comment:
you can also pass `null` to `cookieHeader`, so the type should be `string | object | null`.

https://github.com/reactivestack/cookies/blob/6c7303ccea2ebc687b73f1aefa687037dbb623f9/packages/universal-cookie/src/Cookies.js#L73-L77

Reply:
Done. Thanks for reviewing!